src/MgvOfferTaking.sol

```diff
@@ -467,55 +508,60 @@ abstract contract MgvOfferTaking is MgvHasOffers {
 
   /* ## Execute */
   /* Execution of the offer will be attempted with volume limited by the offer's advertised volume.
-     NB: The caller must ensure that the ratio of the offer is low enough; This is not checked here.
+     **Warning**: The caller must ensure that the price of the offer is low enough; This is not checked here.
 
      Summary of the meaning of the return values:
     * `gasused` is the gas consumed by the execution
     * `makerData` is the data returned after executing the offer
-    * `mgvData` is an [internal Mangrove status code](#MgvOfferTaking/statusCodes).
+    * <a id="MgvOfferTaking/internalStatusCodes"></a>`mgvData` is a status code internal to `execute`. It can hold [any value that `mgvData` can hold](#MgvOfferTaking/statusCodes). Within `execute`, it can additionally hold the following values:
```

Comment:
```suggestion
    * <a id="MgvOfferTaking/internalStatusCodes"></a>`internalMgvData` is a status code internal to `execute`. It can hold [any value that `mgvData` can hold](#MgvOfferTaking/statusCodes). Within `execute`, it can additionally hold the following values:
```